src/session_impl.cpp

```diff
@@ -546,10 +546,6 @@ namespace aux {
 			std::bind(&session_impl::on_lsd_announce, this, _1));
 		TORRENT_ASSERT(!ec);
 
-#ifndef TORRENT_DISABLE_DHT
-		update_dht_announce_interval();
```

Comment:
I might be wrong, but for what I can see, this call in `init` is called by `start_session` and that method is only called if a new `session_impl` is created. How this method can get pass `if (!m_dht)`